### ISSUE TITLE
isherm incorrectly returning True for sigmam() in 3.2.0.dev-

### ISSUE THREAD

**Issue by michaelhush** [qutip/qutip#482]
Hey,

It seems the sigmam() operator incorrectly returns True when tested with isherm.

I think this is only an issue in the developer version 3.2.0.dev- (it does not happen with the stable release)

The following code reproduces the issue:
```python
import qutip as qu

print("My version:")
print(qu.version.version)

mysigmam = qu.basis(2,1) * qu.basis(2,0).dag()
print("My sigmam:")
print(mysigmam)
print("is hermitian?:")
print(mysigmam.isherm)

print("Qutip sigmam:")
print(qu.operators.sigmam())
print("is hermitian?:")
print(qu.operators.sigmam().isherm)
```
Which returns, on my machine:
````
My version:
3.2.0.dev-
My sigmam:
Quantum object: dims = [[2], [2]], shape = [2, 2], type = oper, isherm = False
Qobj data =
[[ 0.  0.]
 [ 1.  0.]]
is hermitian?:
False
Qutip sigmam:
Quantum object: dims = [[2], [2]], shape = [2, 2], type = oper, isherm = True
Qobj data =
[[ 0.  0.]
 [ 1.  0.]]
is hermitian?:
True
```

**Comment by nonhermitian** [comment]
Indeed it is. It appears that it got hard coded somewhere. Will fix tonight. Thanks for finding this!

**Comment by nonhermitian** [comment]
This is addressed in Pull #485.
